helper/impls/helper-impl-14/src/main/java/com/linkedin/avroutil1/compatibility/avro14/Avro14Adapter.java

```diff
@@ -240,8 +242,12 @@ public FieldBuilder newFieldBuilder(String name) {
   public Collection<AvroGeneratedSourceCode> compile(
       Collection<Schema> toCompile,
       AvroVersion minSupportedVersion,
-      AvroVersion maxSupportedVersion
+      AvroVersion maxSupportedVersion,
+      CodeGenerationConfig config
   ) {
+    if (StringRepresentation.CharSequence.equals(config.getStringRepresentation())) {
```

Comment:
Is it right?
I assume avro-1.4 only supports `CharSequence`, right?

Reply:
oops. good catch!